services/news/src/index.ts

```diff
@@ -18,6 +18,7 @@ import express, {Application, Request, Response} from 'express';
 
 import {
   AD_SERVER_HOST,
+  DSP_HOST,
```

Comment:
Remove DSP_HOST. See detailed comment in EJS template.

Reply:
Done.